README.md

```diff
@@ -267,6 +267,9 @@ let counterClone = Counter.fromHex(counter.lockingScript.toHex());
 
 ```
 
+## Chain Api
```

Comment:
Change `C/chain` to `C/chained` to avoid confusion. Chain usually means blockchain in our context.